example/src/Examples/API/AnimatedImages.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { ScrollView, useWindowDimensions } from "react-native";
+import {
+  Canvas,
+  useAnimatedImage,
+  AnimatedImage,
+} from "@shopify/react-native-skia";
+
+export const AnimatedImages = () => {
+  const exampleGif = useAnimatedImage(bse64Gif);
+
+  // TODO - fix this, exampleGif is always null on Android currently
+  console.log(exampleGif);
+
+  const { width: wWidth } = useWindowDimensions();
+  const SIZE = wWidth / 3;
+  const S2 = 60;
+  const PAD = (SIZE - S2) / 2;
+
+  return (
+    <ScrollView>
+      <Canvas
+        style={{
+          alignSelf: "center",
+          width: 320,
+          height: 180,
+          marginVertical: PAD,
+        }}
+      >
+        <AnimatedImage
```

Comment:
I don't think we will ever have an AnimatedImage component.
We can support it from `<Image />` but I would like us to tackle this as a second step.

Reply:
Thanks for the feedback, this makes a lot of sense. I've removed the AnimatedImage component and updated the example in my latest commit. When I have some time I'll look into the decoding issues on Android.